onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -4076,6 +4076,16 @@ def aten_index_put(
     return result
 
 
+@torch_op("aten::_unsafe_index_put")
```

Comment:
```suggestion
@torch_op("aten::_unsafe_index_put", trace_only=True)
```

Because it is reusing aten_index_put